nexus/users/migrations/0004_auto_20190723_1943.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 2.2.3 on 2019-07-23 19:43
```

Comment:
Why is this new migration generated here?

Reply:
This is not a new migration actually. It was missed previously when the `is_core_organizer` and `is_volunteer` were edited to allow `blank` values.

Reply:
This migration is not related to Django upgrade. It is just a missed migration from a previous PR.